packages/tools/e2e-tests/src/support/smart-contracts/faucet/deploy/fundGasStation.ts

```diff
@@ -1,59 +1,26 @@
+import {
+  faucetGasStation,
+  sender00Account,
+} from '@constants/accounts.constants';
+import { InitialFunding } from '@constants/amounts.constants';
+import { transferFunds } from '@helpers/client-utils/transfer.helper';
 import type {
   ChainwebChainId,
   ICommandResult,
 } from '@kadena/chainweb-node-client';
-import { createSignWithKeypair } from '@kadena/client';
-import { transfer } from '@kadena/client-utils/coin';
-import {
-  DEVNET_GENESIS,
-  DOMAIN,
-  GAS_STATION,
-  InitialFunding,
-  NETWORK_ID,
-} from './constants';
 
 export const fundGasStation = async ({
   chainId,
   upgrade,
 }: {
   chainId: ChainwebChainId;
   upgrade: boolean;
-}): Promise<ICommandResult | string> => {
-  if (NETWORK_ID !== 'fast-development') {
-    return 'Only needs to happen on Devnet, funding happens differently on Testnet.';
-  }
-
-  if (upgrade) {
-    return 'The step "fundGasStation" is skipped for upgrades';
-  }
-
-  const transferTask = await transfer(
-    {
-      sender: {
-        account: DEVNET_GENESIS.accountName,
-        publicKeys: [DEVNET_GENESIS.publicKey],
-      },
-      receiver: GAS_STATION,
-      amount: `${InitialFunding}`,
-      gasPayer: {
-        account: DEVNET_GENESIS.accountName,
-        publicKeys: [DEVNET_GENESIS.publicKey],
-      },
-      chainId,
-    },
-    {
-      host: ({ networkId, chainId }) =>
-        `${DOMAIN}/chainweb/0.0/${networkId}/chain/${chainId}/pact`,
-      defaults: { networkId: NETWORK_ID },
-      sign: createSignWithKeypair([
-        {
-          publicKey: DEVNET_GENESIS.publicKey,
-          secretKey: DEVNET_GENESIS.privateKey,
-        },
-      ]),
-    },
+}): Promise<ICommandResult> => {
+  const transfer = await transferFunds(
+    sender00Account,
+    faucetGasStation,
+    InitialFunding,
+    '0',
```

Comment:
Only on chain `'0'`?

Reply:
Good catch, this should be the `chainId`